src/Lecture1.hs

```diff
@@ -50,7 +51,8 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Num a => a -> a -> a
+sumOfSquares x y = x*x + y*y -- x^2 + y^2 wouldn't compile?
```

Comment:
`x ^ 2 + y ^ 2` compiles but it produces warnings because numbers are polymorphic and the `(^)` operator allows different types of base and power.